Specs/Indexer.txt

```diff
@@ -0,0 +1,5 @@
+A method of moving the balls from the indexer to the cataput
+
+Needs to move the balls from the intake to the cataput quickly
+Needs a dejammer
+Needs a colour sensor to id the colour of the ball being indexed
```

Comment:
Indexer will not have a color sensor, the catapult will.

Reply:
Changed